### DISCUSSION TITLE
Sharding | System Design

### DISCUSSION THREAD

**Discussion by giscus** [karanpratapsingh/portfolio#57] [Comments]
# Sharding | System Design

In this tutorial, we will discuss Sharding

https://www.karanpratapsingh.com/courses/system-design/sharding

**Comment by rachitmanit** [comment]
"""
Security: Helps improve the system's security by storing sensitive and non-sensitive data in different partitions. This could provide better manageability and security to sensitive data.
"""

I think doesn't hold true. Sharding is all about dividing tables rows and keeping them in different servers of same kind. Within a table usually one can not find some rows that require high security and others don't. If there is any such cases, they must be divided into two different logical tables.

Let me know if there is any such strong use cases where above holds true.
